pubspec.yaml

```diff
@@ -95,6 +95,10 @@ dependencies:
   cached_network_image: ^3.2.3
   flutter_dotenv: ^5.0.2
   equatable: ^2.0.5
+  image_picker: ^1.1.2
+  flutter_svg: ^2.0.16
+  hive: ^2.2.3
```

Comment:
* Not an important point but as we add hive_flutter dependency we don't need to add hive.
Because hive_flutter uses hive dependency internally. i.e. hive is automatically a dependency.